packages/sdk/typescript/human-protocol-sdk/test/escrow.test.ts

```
@@ -0,0 +1,1368 @@
+
+      const balance = await escrowClient.getBalance(escrowAddress);
+
+      expect(balance).toEqual(amount);
+      expect(escrowClient.escrowContract.getBalance).toHaveBeenCalledWith();
+    });
+
+    test('should throw an error if the getBalance fails', async () => {
+      const escrowAddress = ethers.constants.AddressZero;
+
+      escrowClient.escrowFactoryContract.hasEscrow.mockReturnValue(true);
+      escrowClient.escrowContract.getBalance.mockRejectedValueOnce(new Error());
+
+      await expect(escrowClient.getBalance(escrowAddress)).rejects.toThrow();
+
+      expect(escrowClient.escrowContract.getBalance).toHaveBeenCalledWith();
+    });
+  });
+
+  describe('getManifestUrl', () => {
+    test('should throw an error if escrowAddress is an invalid address', async () => {
+      const escrowAddress = FAKE_ADDRESS;
+
+      await expect(escrowClient.getManifestUrl(escrowAddress)).rejects.toThrow(
+        ErrorInvalidEscrowAddressProvided
+      );
+    });
+
+    test('should throw an error if an hasEscrow return false', async () => {
+      const escrowAddress = ethers.constants.AddressZero;
+
+      escrowClient.escrowFactoryContract.hasEscrow.mockReturnValue(false);
+
+      await expect(escrowClient.getManifestUrl(escrowAddress)).rejects.toThrow(
+        ErrorEscrowAddressIsNotProvidedByFactory
+      );
+    });
+
+    test('should successfully getManifestUrl escrow', async () => {
+      const escrowAddress = ethers.constants.AddressZero;
+      const url = FAKE_URL;
+
+      escrowClient.escrowFactoryContract.hasEscrow.mockReturnValue(true);
+      escrowClient.escrowContract.manifestUrl.mockReturnValue(url);
+
+      const manifestUrl = await escrowClient.getManifestUrl(escrowAddress);
+
+      expect(manifestUrl).toEqual(url);
+      expect(escrowClient.escrowContract.manifestUrl).toHaveBeenCalledWith();
+    });
+
+    test('should throw an error if the getManifestUrl fails', async () => {
```

fix writing:
'should throw an error if getManifestUrl fails'